### ISSUE TITLE
kristallikallot

### ISSUE THREAD

**Issue by Finto-ehdotus** [Finto-ehdotus/YSE#9711]
**Käsitteen tyyppi**

CONCEPT

**Ehdotettu termi suomeksi**

kristallikallot

**Ehdotettu termi ruotsiksi**

kristall(döds)skallar

**Ehdotettu termi englanniksi**

crystal skulls

**Tarkoitusta täsmentävä selite**

ihmisen pääkalloa esittävä veistos, joka on kaiverrettu läpinäkyvään tai vaaleaan kvartsiin

**Perustelut ehdotukselle**

Tarvitaan kuvailussa

**Ehdotettu yläkäsite YSOssa (LT)**

 [taikaesineet](http://www.yso.fi/onto/yso/p1017)

**Ehdotetut temaattiset ryhmät**

 [92 Kuvataide. Maalaustaide. Grafiikka. Kuvanveisto](http://www.yso.fi/onto/yso/p26569)

**Vaihtoehtoiset termit**

**Alakäsitteet (RT)**

**Assosiatiiviset (RT)**

**Vastaava käsite muussa sanastossa**

 [LCSH](https://id.loc.gov/authorities/subjects/sh89006376.html), [Wikipedia](https://fi.wikipedia.org/wiki/Kristallikallo)

**Aineisto jonka kuvailussa käsitettä tarvitaan (esim. nimeke tai URL)**

Kristallimatriisi ja kristallikallojen sanoma / Marjut Moisala.

**Ehdottajan organisaatio**

**Termiehdotus Fintossa:** [kristallikallot](http://finto.fi/yse/fi/page/Y509711)

**Comment by mirmaid** [comment]
Eli onko kyse tästä: _Crystal skulls are human skull hardstone carvings made of clear or milky white quartz (also called "rock crystal"), claimed to be pre-Columbian Mesoamerican artifacts by their alleged finders; however, these claims have been refuted for all of the specimens made available for scientific studies._

**Comment by mariaforsen** [comment]
Kyllä, kivestä tehtyjä kalloja, ovat tietysti aivan oikeita esineitä, joita jotkut sitten myös pitävät "yliluonnollisina"

LCSH:ssa termin (crystal skulls) yläkäsitteitä ovat Carving (Decorative arts) ja Occultism

Wikipedia: "Kristallikallo on ihmisen pääkalloa esittävä veistos, joka on kaiverrettu läpinäkyvään tai vaaleaan kvartsiin. Kristallikallot tulivat tunnetuiksi kertomusten ja uskomusten aiheiksi 1800-luvun länsimaissa. Niiden väitettiin olevan peräisin Väli-Amerikan korkeakulttuureilta, ajalta ennen Amerikan valloitusta. Niihin liitettiin ennustuksia, historiaa ja yliluonnollisia ilmiöitä. Yhdenkään kristallikallon ei ole kuitenkaan voitu osoittaa olevan peräisin väitetyltä ajanjaksolta tai alueelta."

**Comment by mirmaid** [comment]
Tämä kirja https://www.bod.fi/kirjakauppa/kristallimatriisi-ja-kristallikallojen-sanoma-marjut-moisala-9789528036821

Olemmeko kuitenkaan valmiita luopumaan vanhasta mielen matriisista 🤔

**Comment by mariaforsen** [comment]
Nämä ovat fyysisiä esineitä ja ovat jo LCSH:ssa ja SAOssakin, meidän ei siis tarvitse millään tavalla ottaa kantaa niihin liittyviin uskomuksiin :)

**Comment by mirmaid** [comment]
^ Mainittu kirja on kuitenkin ainoa viite Melindassa, eikä Finnassakaan ole muuta. Ovatko kristallikallot jotenkin tärkeämpiä kuin muista materiaaleista valmistetut, muuten kuin noihin taikaesineuskomuksiin liittyen?
